BankManagementSystem/Program.cs

```diff
@@ -140,7 +140,12 @@ static void Main(string[] args)
                          password = Console.ReadLine();
                         Console.WriteLine("Enter the amount to withdraw");
                         amount = Convert.ToDouble(Console.ReadLine());
-                        accountControllerObj.Withdraw(accountNumber, password, amount);
+                        if (customerObject.AccountDetails.Balance <= amount)
+                            accountControllerObj.Withdraw(accountNumber, password, amount);
+                        else
+                        {
+                            Console.WriteLine($"Please enter the amount less than  the balance amount {customerObject.AccountDetails.Balance}");
```

Comment:
Add one condition to check for negative values

Reply:
please check it now